src/ts/components/subtitlevtt.ts

```diff
@@ -0,0 +1,157 @@
+import { SubtitleRegionContainer, SubtitleLabel } from './subtitleoverlay';
```

Comment:
As this is no component please move one folder level up. (Or think about introducing a helper folder)